projects/netgrif-components-core/src/lib/panel/task-panel/abstract-task-panel.component.ts

```diff
@@ -406,7 +406,7 @@ export abstract class AbstractTaskPanelComponent extends AbstractPanelWithImmedi
                 }
                 return {value: 'low', icon: 'south', type: 'meta'};
             case TaskMetaField.USER:
-                return {value: task.user ? task.user.fullName : '', icon: 'account_circle', type: 'meta'};
+                return {value: task.userId ? task.userId : '', icon: 'account_circle', type: 'meta'};
```

Comment:
This isn't an equivalent replacement. Before the meta field displayed the name of the user who has the task assigned, now it only displays its ID. In my opinion, this is not how it should be.

Reply:
This was needed, because we do not have user on task for now, only userId. After the User refactor PR will be merged, we will refactor this with ActorRef, so this is only temporary. I added todo to it.